docs/Hooks.md

```diff
@@ -284,6 +284,9 @@ fastify.addHook('onRoute', (routeOptions) => {
   routeOptions.prefix
 })
 ```
+
+If you are authoring a plugin and you need to customize application routes, like modifying the options or adding new route hooks, this is the right place.
```

Comment:
What do you think about to provide a code example here?

Reply:
Yes, I can and I will. :)

Reply:
Done!